src/tests/unit/tests/injected/visualization/drawer-utils.test.ts

```diff
@@ -48,6 +48,7 @@ describe('getBoundingClientRectIncludingChildren', () => {
         expect(result.right).toBe(15);
     });
 
+    // tslint:disable-next-line: typedef
```

Comment:
>// tslint:disable-next-line: typedef [](start = 4, length = 36)

I think you could avoid this and also make the individual tests less redundant if you had this helper return type `Element` and did the type punning here instead of in a repeated `as any` in each test case

Reply:
Per offline discussion, deferring this for a separate PR